### ISSUE TITLE
App crashes immediately after opening

### ISSUE THREAD

**Issue by mapleroyal** [Automattic/simplenote-electron#3128]
This on on vanilla Arch Linux using the appimage from here. On both 2.21 and 2.20. Haven't tried older versions, and this is the first time I've ever tried this app.

Behavior: opening the appimage, a simplenote window appears for about 1 second, before disappearing.

Expected behavior: the simplenote window is the app, so it should continue to remain open and allow me to use it

Special note: I did try the version from apprepo before trying the official ones from here. Those work, and I was able to get an account set up and use the app, but I would prefer the official appimage (which also happens to be a more current version, where the apprepo one is at like 2.17).

**Comment by mapleroyal** [comment]
More details: I downloaded the official 2.7.1, which is the same as the version at apprepo. It works as expected. Obviously I don't want to stay on an old version, I selected the option from the popup about downloading and installing the newest version.

The window disappeared for a second, and a new window popped up (I assume the updated version), but it closed immediately, as the new version does.

Additionally, the update process seems to have moved/deleted my 2.7.1 appimage, and I don't know what it's done with it or where it's put the updated version. I'd like to not have files all over my system, and a search doesn't bring up the 2.7.1 version or the version it updated to, so if someone could point out the directory those are in that would be awesome.
